packages/core/src/components/StepList/StepLink.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class StepLink extends React.PureComponent {
+  handleClick(event) {
+    if (this.props.onEnterStep) {
```

Comment:
This was a decision made after discussions and much deliberations. The problem with `onClick` is that it's ambiguous. What's being clicked? And this function is only for routing that is triggered by clicking certain elements. I'm willing to budge on this, but you'll have to argue your case.

Reply:
I didn't realize you originally just meant this one sub-component. That makes sense in this context.

Reply:
At first I didn't want something long-winded because of code formatting, but all the declaration of StepLinks are multi-line anyway. `onStepLinkClick` makes sense. It also wasn't' originally called `StepLink`